doc/user_guide/install.rst

```diff
@@ -14,6 +14,12 @@ see the :ref:`anaconda-install` section.
 To enable context-menu (right-click) shortcut in a chosen folder, use the
 `start_jupyter_cm <https://github.com/hyperspy/start_jupyter_cm>`_ library.
 
+.. note::
+
+    If you want to be notified about new releases, please *Watch (Releases only)* 
+    the `hyperspy repositorium on GitHub <https://github.com/hyperspy/hyperspy/>`_ 
```

Comment:
The standard term is "repository" I think.

Reply:
sure ;-)